apps/teams-test-app/src/components/privateApis/PrivateAPIs.tsx

```diff
@@ -1,4 +1,5 @@
 import {
+  dataLayer,
```

Comment:
Recommend adding a new file into the test app that contains both your core API test and the capability check. 

You can see here I have just updated my test app component to include the standard format for checking the capability: https://github.com/OfficeDev/microsoft-teams-library-js/pull/2260/files#diff-b23c94c513dc820dfc219abbcdf724085cf88df4c9bcc0bc5b07e84009ed9235

Reply:
updated. thanks